docs/docs/cmd/teams/meeting/meeting-create.mdx

```diff
@@ -0,0 +1,241 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# teams meeting create
+
+Create a new online meeting
+
+## Usage
+
+```sh
+m365 teams meeting create [options]
+```
+
+## Options
+
+```md definition-list
+`-s --startTime [startTime]`
+: The start time of the meeting. If not specified, the startTime will be set to the current time.
+
+`-e --endTime [endTime]`
+: The end time of the meeting. If not specified, the endTime will be set to one hour after the startTime.
+
+`--subject [subject]`
+: The subject of the meeting.
+
+`-p, --participants [participants]`
+: A comma-separated list of participant UPNs.
+
+`--organizerEmail [organizerEmail]`
+: The organizer's email address. Requires application permissions.
```

Comment:
Let's put a remark in a Remarks section about the app permissions. That's a better place than here

Reply:
I hope that it is the correct place. Right after  <Global />